custom/icds_reports/management/commands/quarter_school_data.py

```diff
@@ -0,0 +1,89 @@
+
+from datetime import date
+from dateutil import parser
+import csv
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+from custom.icds_reports.models.aggregate import AwcLocation
+from dimagi.utils.chunked import chunked
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+db_alias = get_icds_ucr_citus_db_alias()
+
+
+def _run_custom_sql_script(command):
+    with connections[db_alias].cursor() as cursor:
+        cursor.execute(command)
+        return cursor.fetchall()
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        state_locations = AwcLocation.objects.filter(aggregation_level=1, state_is_test=0).values('state_id',
+                                                                                                  'state_name')
+        final_state_data = {
+            location['state_id']: {
+                'state_name': location['state_name'],
+                36: 0,
+                48: 0,
+                60: 0,
+                72: 0
+            }
+            for location in state_locations
+        }
+
+        query = """
+                    SELECT
+                        ucr.state_id,
+                        ucr.mother_id
+                    FROM  child_health_monthly
+                    INNER JOIN "ucr_icds-cas_static-child_health_cases_a46c129f" ucr
+                    ON (
+                        child_health_monthly.case_id=ucr.doc_id AND
+                        child_health_monthly.supervisor_id=ucr.supervisor_id AND
+                        month>='2020-01-01' AND month<='2020-03-01'
```

Comment:
nit: using <'2020-04-01' would read better I think and won't need this dashboard context of how every thing is a matter of month represented by the first of the month.